.github/workflows/jira-issue-create.yml

```diff
@@ -26,10 +26,10 @@ jobs:
           summary: |
             [SDK - iOS] ${{ github.event.issue.title }}
           description: |
-            ${{ github.event.issue.url}}
+            ${{ github.event.issue.html_url }}
```

Comment:
is this fixing the github url on the Jira ticket description?

Reply:
Yes, this is. This provides the url "github.com..." instead of "api.github.com..."